x-pack/plugins/fleet/public/applications/fleet/sections/agents/agent_list_page/index.tsx

```diff
@@ -357,6 +358,14 @@ export const AgentListPage: React.FunctionComponent<{}> = () => {
       render: (host: string, agent: Agent) => (
         <EuiLink href={getHref('fleet_agent_details', { agentId: agent.id })}>
           {safeMetadata(host)}
+          {agent.type === 'EPHEMERAL' && (
+            <EuiBadge color="secondary" style={{ marginLeft: '1em' }}>
```

Comment:
instead of doing inline CSS, you can wrap the agent name and badge in a `EuiFlexGroup` with `gutter`

Reply:
👍🏻 I tried that (flexgroup with gutter, and two flexitems) but hit an alignment issue. The type badge and status badge were 1px different which is noticable since they're right next to each other. I didn't dig too much since I wasn't certain this was how we wanted to display it, but I'll put it back and push so we can get more 👀